home.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+
+    <!-- custom css -->
+    <link rel="stylesheet" href="style.css">
+
+    <!-- fontawesome cdn -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css"
+        integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+
+    <!-- google font -->
+    <style>
+        @import url('https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700;800&display=swap');
+    </style>
+</head>
+
+<body>
+    <header class="toolbar">
+        <h1>Welcome</h1>
+        <i class="fa-solid fa-bars"></i>
+    </header>
```

Comment:
- Good job wrapping the toolbar section in a `<header>` tag 💪🏽 

- I think your logo (Welcome) should have a link (`<a>` tag). This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.

- Also, I think your hamburger menu should be in a `nav` tag is for semantics reason, and the right position for convention and user experience for mobile devices.

I would suggest you refactor your code(line 25 to 28) to look something like this 👇🏽 
```
    <header class="toolbar">
        <h1><a href="#">Welcome</a></h1>
        <nav>
            <i class="fa-solid fa-bars"></i>
        </nav>
    </header>
```

What do you think about this suggestion? 🤔

Reply:
Thanks for the suggestions. Let me fix it right away.🙏